### ISSUE TITLE
[Discussion] Documentation vs wiki

### ISSUE THREAD

**Issue by chernjie** [tj/git-extras#358]
### TL;DR
Documentation should be in the same git repo as it's main project

### Long version
I understand `git-extras` have recently moved the long winded contents of `Readme.md` into Github's wiki. It is a good thing as `Readme.md` has become too long and at some point has become difficult to maintain.

Having said that, I do think that the `HEAD` of each wiki ref should stay in sync with the `HEAD` of each pull request. This ensures that documentations are not left out and encourage script discovery.

@hemanth has been tirelessly advising pull requesters to update the wiki on every successful review. Perhaps we could be suggested that unless documentations are updated, the PR will not be merged?

### Proposed solution (simple)
Since Github's wiki is also a Git repo, we can merge it into the main repo without losing history or `refheads`. Effectively retiring Github wiki.

What are your thoughts on this?

**Comment by hemanth** [comment]
I had been thinking about it for some time now, the proposed solution looks neat, lets go for it!

**Comment by chernjie** [comment]
Awesome I will submit a pull request

**Comment by spacewander** [comment]
@chernjie 
:+1:

**Comment by chernjie** [comment]
Pull request created at #359

**Comment by raviranjananalist** [comment]
For Expert 5 Packers and Movers Hyderabad go to  
http://www.expert5th.in/packers-and-movers-hyderabad/
For Expert 5 Packers and Movers Bangalore go to 
http://www.expert5th.in/packers-and-movers-bangalore/
For Expert 5 Packers and Movers Mumbai go to 
http://www.expert5th.in/packers-and-movers-mumbai/

**Comment by raviranjananalist** [comment]
For Expert 5 Packers and Movers Gurgaon go to 
http://www.expert5th.in/packers-and-movers-gurgaon/
For Expert 5 Packers and Movers Delhi go to 
http://www.expert5th.in/packers-and-movers-delhi/
For Expert 5 Packers and Movers Pune go to  
http://www.expert5th.in/packers-and-movers-pune/

**Comment by raviranjananalist** [comment]
For Expert 5 Packers and Movers Chennai go to 
http://www.expert5th.in/packers-and-movers-chennai/
For Expert 5 Packers and Movers Noida go to 
http://www.expert5th.in/packers-and-movers-noida/
For Expert 5 Packers and Movers Thane go to 
http://www.expert5th.in/packers-and-movers-thane/
For Expert 5 Packers and Movers Navi Mumbai go to 
http://www.expert5th.in/packers-and-movers-navimumbai/
For Expert 5 Packers and Movers Ghaziabad go to 
http://www.expert5th.in/packers-and-movers-ghaziabad/
For Expert 5 Packers and Movers Faridabad go to 
http://www.expert5th.in/packers-and-movers-faridabad/

**Comment by raviranjananalist** [comment]
For Expert 5 Packers and Movers Bangalore go to 
http://www.expert5th.in/packers-and-movers-bangalore/
For Expert 5 Packers and Movers Pune go to  
http://www.expert5th.in/packers-and-movers-pune/
For Expert 5 Packers and Movers Mumbai go to 
http://www.expert5th.in/packers-and-movers-mumbai/
